src/main/java/org/macrulez/utils/raml2swagger/RAMLtoSwagger.java

```diff
@@ -115,44 +129,43 @@ public String getBasePath() {
     //Put all the API basic info
     private void getAPIInfo() throws JSONException {
         JSONObject info = new JSONObject();
-        if(!raml.getTitle().isEmpty())
+
+        if (!raml.getTitle().isEmpty())
             info.put(API_TITLE_PARAM_KEY, raml.getTitle());
 
-        if(raml.getVersion() != null)
+        if (raml.getVersion() != null)
             info.put(API_VERSION_PARAM_KEY, raml.getVersion());
 
+        if (raml.getDocumentation() != null) {
+            info.put(DESCRIPTION_PARAM_KEY, raml.getDocumentation().stream().map(i -> i.getTitle() + " - " + i.getContent()).collect(Collectors.joining(" | ")));
+        }
+
+
         swaggerJSON.put(API_INFO_PARAM_KEY, info);
 
         //Get the baseURI
         URL url = null;
         try {
             url = new URL(raml.getBaseUri());
         } catch (MalformedURLException e) {
-            e.printStackTrace();
+            LOGGER.error("Error getting baseUri", e);
         }
 
         //Incase the schemes is mentioned in the RAML, use it. Else parse it from the API URL
-        if(raml.getProtocols().size() > 0) {
-            List<String> protoList = new ArrayList<>();
-            for(Protocol p : raml.getProtocols()) {
-                protoList.add(p.toString().toLowerCase());
-            }
-            swaggerJSON.put(SCHEMES_PARAM_KEY, protoList);
+        if (raml.getProtocols().size() > 0) {
+            swaggerJSON.put(SCHEMES_PARAM_KEY, raml.getProtocols().stream().map(p -> p.toString().toLowerCase()).collect(Collectors.toList()));
```

Comment:
I would suggest to add a line of comment for the users to understand easily in places where code seems pretty complex.

Reply:
Sure. This is the Java stream API introduced in Java8 to simplify work with collections. Will comment.

Reply:
Why dont we go with the earlier one so as to ensure compatibility for Java 7?

Reply:
Sorry for the late reply. I thought this is a standalone tool, not a library therefore compatibility is not a question.

Java 7 is out of life and no longer supported. Java 9 is already out. Whereas I embrace backwards compatibility, sometimes there's no reason for it. Java 8+ comes with lots of performance benefit and a functional interface.The map() function is part of this and reduces unnecessary verbosity.

At the end of the day this id your project and you decide how you want to do it, all I'm saying that Java7 should not be a goal for this project in my opinion.

Reply:
Actually, even I created it as a standalone tool. But, I am not sure whether users are interested in it as a standalone tool or as a lib. So, for now, I will merge your request. And if the users' needs change, we will either rollback or add that as a submodule. 
So, final check..are all the test cases' outputs equal with the outputs of the previous code?